composer.json

```diff
@@ -22,8 +22,8 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "ext-tokenizer": "*",
-        "composer/composer": "^1.7 || ^2.0",
-        "friendsofphp/php-cs-fixer": "^2.16.1",
+        "composer/composer": "^2.0-dev",
+        "friendsofphp/php-cs-fixer": "^3.0.x-dev",
```

Comment:
:thinking: Theses dev version, to avoid problem with xdebug-handler, right ?

Reply:
note that the 2.0.14 version of composer released today use xdebug-handler v2 :slightly_smiling_face:

Reply:
Yes, at the moment I opened this PR, the composer was still using the 1.1 version. Now I can update it to the new version. Thanks!